### DISCUSSION TITLE
Bell IPTV

### DISCUSSION THREAD

**Discussion by Qlogic711** [fabianishere/udm-iptv#150] [Q&A]
Need a little help, everything was working perfectly until today. I got an email saying bell would be doing upgrades in my area last night, but not sure if this would affect it

but now TV doesn't work

I am getting this error

root@:~# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth9
WAN VLAN: 36 (dev iptv)
WAN DHCP: true (options "-n -o")
WAN Ranges: 10.2.0.0/16 10.125.0.0/16
LAN Interfaces: br40
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
=== Service Logs ===
Jul 25 16:37:06  udm-iptv[3314]: Sending discover...
Jul 25 16:37:10  udm-iptv[3314]: udhcpc failed to get a DHCP lease
Jul 25 16:37:10  udm-iptv[3314]: No lease, forking to background
Jul 25 16:37:10  udm-iptv[3314]: NATing IPTV network ranges (if necessary)
Jul 25 16:37:10  udm-iptv[3314]: Setting up igmpproxy
Jul 25 16:37:10  udm-iptv[3314]: Starting igmpproxy
Jul 25 16:37:10  igmpproxy[3314]: There must be at least 1 Vif as upstream.
Jul 25 16:37:10  systemd[1]: udm-iptv.service: Main process exited, code=exited, status=255/EXCEPTION
Jul 25 16:37:10  systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Jul 25 16:37:10  systemd[1]: udm-iptv.service: Consumed 114ms CPU time.

and no IP when running 

ip -4 addr show dev iptv

any thoughts?

**Comment by Qlogic711** [comment]
I am connected to the SFP+1 port which should be eth9

**Reply by Qlogic711** [reply]
hooked up my old edge router and get an IP no problem. am I missing something?

**Comment by dcrompto** [comment]
@Qlogic711 giving this a bump, i was using the app so much didn't see my iptv break now getting the same error. Were you able to get this working?

**Reply by Qlogic711** [reply]
No I haven't. That's why I ended up putting my old edge router back. I run both the udm and the edge router
